CatAnalyzer/test/customise_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
> create-batch --jobName TT_powheg --fileList ifdjfidfd/CATTools/CatAnalyzer/data/dataset_TT_powheg.txt --maxFiles 20 --cfg run_TtbarBbbarDiLeptonAnalyzer_cfg.py --transferDest jdfidfjieeef/v7-4-6v2/TT_powheg --customise customise_cff.py

current, I am using the code.

perhaps, we can set up as like followings.
Base_cfg.py and TtbarBbbar_cff.py
++Base_cfg.py : basic configuration for load
++TtbarBbbar_cff.py : load for ttbb.